cognite_toolkit/_cdf_tk/loaders/_base_loaders.py

```diff
@@ -267,20 +267,16 @@ def _split_ids(cls, ids: T_ID | int | SequenceNotStr[T_ID | int] | None) -> tupl
         raise ValueError(f"Invalid ids: {ids}")
 
     def load_resource_file(
-        self, filepath: Path, ToolGlobals: CDFToolConfig, skip_validation: bool
+        self, filepath: Path, ToolGlobals: CDFToolConfig, is_dry_run: bool = False
```

Comment:
Renamed this one to make it more aligned with the CLI such that it is easier to remember what it is doing.